docs/quickstart.md

```diff
@@ -2,6 +2,12 @@
 
 ## Deploying DataHub
 
+:::note
+
+This is not recommended for production use. This is supposed to be used for small POCs only or for playing around with DataHub to get familiar with the features. Please see [Deploying with Kubernetes](deploy/kubernetes.md) for the recommended production setup.
```

Comment:
Not so sure we need such a bold warning. Some people actively are using this for production... what do we tell those folks?